src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -305,6 +312,30 @@ export class CellModel extends Disposable implements ICellModel {
 		this._preventNextChartCache = true;
 	}
 
+	private attachImageFromSource(newSource: string | string[]): string | string[] {
+		if (!Array.isArray(newSource) && this.isValidBase64OctetStream(newSource)) {
+			let results;
+			while ((results = validBase64OctetStreamRegex.exec(newSource)) !== null) {
+				let imageName = this.addAttachment(results[1], results[0], 'image.png');
+				newSource = newSource.replace(validBase64OctetStreamRegex, `attachment:${imageName}`);
+			}
+			return newSource;
+		}
+		return newSource;
+	}
+
+	private getUniqueImageName(imgName?: string, imgExtension?: string): string {
```

Comment:
A better name would be `getUniqueAttachmentName` since that's what it's actually checking